piccolo/columns/base.py

```diff
@@ -599,18 +600,38 @@ def _validate_choices(
 
         return True
 
-    def is_in(self, values: list[Any]) -> Where:
-        if len(values) == 0:
-            raise ValueError(
-                "The `values` list argument must contain at least one value."
-            )
+    def is_in(self, values: Union[Select, list[Any]]) -> Where:
```

Comment:
I'd like to change this up a bit, so rather than passing a `Select` into `is_in` we directly pass the `Querystring`. 

It makes it a bit more flexible.